src/nvim/option.c

```diff
@@ -301,6 +301,15 @@ static char *(p_cot_values[]) = {"menu", "menuone", "longest", "preview",
 # include "option.c.generated.h"
 #endif
 
+static void set_runtimepath_default(void)
+{
+  garray_T rtp_ga;
+  ga_init(&rtp_ga, (int)sizeof(const char *), 1);
+  GA_APPEND(const char *, &rtp_ga, get_user_conf_dir());
+  GA_APPEND(const char *, &rtp_ga, concat_fnames(get_user_conf_dir(), "after", true));
+  set_string_default("runtimepath", ga_concat_strings(&rtp_ga));
```

Comment:
Would you suggest that I precomute the length of the rtp string like this?
https://github.com/ZyX-I/neovim/commit/b4331ace2b47260867ef619840fa38331a1f2f41